src/dotnet/CarbonAware/CarbonAwareVariablesConfiguration.cs

```diff
@@ -5,6 +5,38 @@ namespace CarbonAware;
 /// </summary>
 public class CarbonAwareVariablesConfiguration
 {
+    /// <summary>
+    /// The Key containing the configuration values.
+    /// </summary>
     public const string Key = "CarbonAwareVars";
+
+    private string webApiRoutePrefix = null;
+
+    /// <summary>
+    /// Gets or sets the the carbon intensity data source to use.
+    /// </summary>
     public string CarbonIntensityDataSource { get; set; }
+
+    /// <summary>
+    /// Gets or sets the route prefix to use for all web api routes.  Defaults to null.
+    /// </summary>
+    public string WebApiRoutePrefix
```

Comment:
Is string the right type to use here? Can we use something more structured?

Reply:
Not really.  This comes from json and really reflects a common path you want ALL routes to have.  For example, /sci-scores becomes /george/sci-scores if the WebApiRoutePrefix configuration value is set to "george".  Because you're only adding a fragment of a path, there's not really anything besides a string that will work.